x/market/genesis.go

```
@@ -48,22 +48,35 @@
 	for _, record := range data.Leases {
 		key := keys.MustLeaseKey(keys.LeaseStateToPrefix(record.State), record.ID())
+		revKey := keys.MustLeaseReverseKey(keys.LeaseStateToPrefix(record.State), record.ID())
 
 		if store.Has(key) {
-			panic(fmt.Errorf("market genesis leases init. order id %s: lease exists", record.ID()))
+			panic(fmt.Errorf("market genesis leases init. lease id %s: lease exists", record.ID))
 		}
 
-		store.Set(key, cdc.MustMarshal(&record))
+		if store.Has(revKey) {
+			panic(fmt.Errorf("market genesis leases init. reverse key for lease id %s: lease exists", record.ID))
```

golangci-lint (1.64.8)

SA5009: Printf format %s has arg #1 of wrong type func() github.com/akash-network/akash-api/go/node/market/v1beta4.LeaseID

(staticcheck)